trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    name = @nameable.correct_name
+    name.length > 10 ? name[0...10] : name
```

Comment:
- [ ] to show a name with a maximum of 10 characters your code should be ` name[0...9]` the one you have currently will check for a maximum of 11 characters